### DISCUSSION TITLE
cannot run locally

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#6549] [Archive]
try to use this guide: https://docs.directus.io/contributing/running-locally/

on 4 step (build) got error:
![image](https://user-images.githubusercontent.com/36981278/123704657-90aa1580-d86e-11eb-9361-cb2ce0f50232.png)

if try `npm run dev`
got that error:
![image](https://user-images.githubusercontent.com/36981278/123704785-bafbd300-d86e-11eb-8bbb-4987a362c659.png)

use latest node with windows 10 x64, what wrong?

**Comment by rijkvanzanten** [comment]
This is a known problem on windows and has been resolved in https://github.com/directus/directus/pull/6215 👍🏻
